docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+version: "3.9"
+
+services:
+  app:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py migrate &&
+             python manage.py runserver 0.0.0.0:8000"

```

Comment:
The command to run migrations and start the server is correct, but ensure that the `migrate` command completes successfully before starting the server. Consider using a more robust entrypoint script if needed.